### PR TITLE
Narrator Reads mobile chat and More buttons

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Add strings for More button on mobile.
update ControlBarButton to read aria-labels from other strings
# Why

Allows for better narration of the main controls in the CallWIthChat

https://skype.visualstudio.com/SPOOL/_workitems/edit/2779666
# How Tested

Ran locally testing with windows narrator in mobile mode.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

File: change/@internal-react-components-88cf0df0-80da-4850-97cd-484c6bca2969.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (JamesBurnside) [inline change/@internal-react-components-88cf0df0-80da-4850-97cd-484c6bca2969.json:2]: nit: this is a patch change, affects stable build

Comment (JamesBurnside) [reply change/@internal-react-components-88cf0df0-80da-4850-97cd-484c6bca2969.json:2]: (other changefile stays prerelease)

File: packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx [hunk 2]
```diff
@@ -45,6 +45,14 @@ export interface CallWithChatCompositeStrings {
    * Only used with `mobileView` set to `true`.
    */
   moreDrawerSpeakerMenuTitle: string;
+  /**
+   * {@Link CallWithChatComposite} Mobile Control bar More drawer label for narrator
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx:49]: The comment doesn't seem to match the field name / intent. This is also the label that is shown on the UI, right?

File: packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx [hunk 3]
```diff
@@ -45,6 +45,14 @@ export interface CallWithChatCompositeStrings {
...
+   */
+  moreDrawerButtonLabel: string;
+  /**
+   * {@Link CallWithChatComposite} Mobile Control bar More drawer label for narrator
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx:53]: The comment doesn't seem to match intent. This is used in the tooltip that is shown in the UI on hover, right?

**Review by prprabhu-ms — APPROVED** [review]

File: packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx (hunk 2 above)
Comment (dmceachernmsft) [reply packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx:49]: Right, I labelled it in the context I was working on when its really a side effect! will change this to be more appropriate 😅

**Pull request merged**